02-peer-protocol.md

```diff
@@ -3361,35 +3366,24 @@ The sending node:
   - otherwise:
     - MUST NOT set `next_funding_txid`.
   - if `option_splice` was negotiated:
-    - MUST set `your_last_funding_locked` to the txid of the last `splice_locked` it received.
-    - if it never received `splice_locked` for any transaction, but it received `channel_ready`:
-      - MUST set `your_last_funding_locked` to the txid of the channel funding transaction.
-    - otherwise (it has never received `channel_ready` or `splice_locked`):
-      - MUST NOT set `your_last_funding_locked`.
     - if a splice transaction reached acceptable depth while disconnected:
-      - MUST set `my_current_funding_locked` to the txid of the latest such transaction.
-      - MUST send `splice_locked` for that transaction after exchanging `channel_reestablish`.
+      - MUST include `my_current_funding_locked` with the txid of the latest such transaction.
     - otherwise:
-      - MUST set `my_current_funding_locked` to the txid of the last `splice_locked` it sent.
+      - MUST include `my_current_funding_locked` with the txid of the last `splice_locked` it sent.
       - if it never sent `splice_locked` for any transaction, but it sent `channel_ready`:
-        - MUST set `my_current_funding_locked` to the txid of the channel funding transaction.
+        - MUST include `my_current_funding_locked` with the txid of the channel funding transaction.
       - otherwise (it has never sent `channel_ready` or `splice_locked`):
-        - MUST NOT set `my_current_funding_locked`.
+        - MUST NOT include `my_current_funding_locked`.
       - if `my_current_funding_locked` is included:
         - if `announce_channel` is set for this channel:
           - if it has not received `announcement_signatures` for that transaction:
-            - MUST retransmit `channel_ready` or `splice_locked` after exchanging `channel_reestablish`.
-        - if it receives `channel_ready` for that transaction after exchanging `channel_reestablish`:
-          - MUST retransmit `channel_ready` in response, if not already sent since reconnecting.
-        - if it receives `splice_locked` for that transaction after exchanging `channel_reestablish`:
-          - MUST retransmit `splice_locked` in response, if not already sent since reconnecting.
+            - MUST set the `announcement_signatures` bit to `1` in `retransmit_flags`.
+        - otherwise:
+          - MUST set the `announcement_signatures` bit to `0` in `retransmit_flags`.
 
 A node:
   - if `next_commitment_number` is 1 in both the `channel_reestablish` it
-    sent and received and `option_splice` was NOT negotiated:
-    - MUST retransmit `channel_ready`.
-  - if `option_splice` was negotiated and `your_last_funding_locked` is not
-    set in the `channel_reestablish` it received:
+    sent and received:
```

Comment:
Not entirely familiar with this, but is reverting this a problem for https://github.com/lightning/bolts/pull/1160/commits/f9fd539db6cc6f3e532fdc8cc1ebe8eb1a8fd717#r1975115655 ?